test/unit/manipulation.js

```diff
@@ -2297,7 +2297,7 @@ testIframe(
 // Skip the the test if we are not in localhost but make sure we run
 // it in Karma.
 QUnit[
-	jQuery.ajax && ( window.__karma__ || location.hostname === "localhost" ) ?
+	jQuery.ajax && ( ( window.__karma__ && location.hostname !== "bs-local.com" ) || location.hostname === "localhost" ) ?
```

Comment:
We want this test running in Karma and this condition creates a risk of us skipping it completely. Can you elaborate? Whatever we need to exclude, we need to do it in a safer way.